### PR TITLE
Merge september

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-base/build.gradle [hunk 1]
```diff
@@ -93,6 +93,8 @@ subprojects {
         imports {
             mavenBom 'com.microsoft.azure:azure-toolkit-libs:0.13.0-SNAPSHOT'
             mavenBom 'com.microsoft.azure:azure-toolkit-ide-libs:0.13.0-SNAPSHOT'
+            mavenBom 'com.microsoft.azure:azure-toolkit-libs:0.12.1'
+            mavenBom 'com.microsoft.azure:azure-toolkit-ide-libs:0.12.1'
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-base/build.gradle]: both `0.13.0-SNAPSHOT` and `0.12.1` are here.

File: Utils/pom.xml [hunk 2]
```diff
@@ -32,14 +32,15 @@
     <name>${project.artifactId}-${project.version}</name>
     <properties>
         <azuretool.version>3.58.0-SNAPSHOT</azuretool.version>
+        <azure.version>1.41.2</azure.version>
```
Comment (wangmingliang-ms) [inline Utils/pom.xml:35]: what is this used for?

File: Utils/hdinsight-node-common/pom.xml [hunk 3]
```diff
@@ -232,6 +232,7 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure-mgmt-storage</artifactId>
+            <version>${azure.version}</version>
```
Comment (wangmingliang-ms) [inline Utils/hdinsight-node-common/pom.xml:235]: why is this not needed before?

File: Utils/pom.xml (hunk 2 above)
Comment (wangmingliang-ms) [inline Utils/pom.xml:35]: why is this not needed before?

File: Utils/hdinsight-node-common/pom.xml (hunk 3 above)
Comment (andxu) [reply Utils/hdinsight-node-common/pom.xml:235]: I remove track1: azure-mgmt-storage from maven repository

File: Utils/pom.xml (hunk 2 above)
Comment (andxu) [reply Utils/pom.xml:35]: I remove track1 sdk from maven repository

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/appservice/serviceplan/ServicePlanComboBox.java [hunk 4]
```diff
@@ -60,7 +60,7 @@ public void setSubscription(Subscription subscription) {
         // Clean up app service plan cache when switch subscription
         // todo: leverage event hub to update resource cache automatically
         try {
-            CacheManager.evictCache("appservcie/{}/plans", subscription.getId());
```
Comment (wangmingliang-ms) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/appservice/serviceplan/ServicePlanComboBox.java:63]: is the corresponding `@Cacheable` fixed?

**Review by wangmingliang-ms — APPROVED** [review]

**Pull request merged**